### ISSUE TITLE
Prismatic Ring with Evasion affix is showing evasion as an implicit

### ISSUE THREAD

**Issue by Reithan** [PoE-Overlay-Community/PoE-Overlay-Community-Fork#87]
# 🐞 bug report

**Describe the bug**
Prismatic Ring with Evasion affix is showing evasion as an implicit:

**Screenshots**
![image](https://user-images.githubusercontent.com/7475701/85342913-7fcb0500-b4a0-11ea-8950-54ea6276bce9.png)

## 🌍  Your Environment

- OS: Win10 x64
- Version: 0.7.1 full install
- PoE: 3.11 PC Harvest SC

**Comment by WhiteFang5** [comment]
Can you CTRL+C the item and paste the result in a new comment please?

**Comment by Reithan** [comment]
Rarity: Rare
Chimeric Gyre
Prismatic Ring
--------
Requirements:
Level: 33
--------
Item Level: 47
--------
+10% to all Elemental Resistances (implicit)
--------
5% increased Cast Speed
+73 to Evasion Rating
+46 to maximum Mana
+11% to all Elemental Resistances

**Comment by WhiteFang5** [comment]
From the looks of the ctrl+C data and the screenshot, it seems that all stats are being transformed to pseudo stats, except for the evasion rating.  
Because pseudo stats always appear at the bottom (after implicit and explicit stats), it only looks like the evasion rating is an implicit.  
  
TL;DR: It's not actually a bug and all works as intended.

**Comment by Reithan** [comment]
Ah, I see, so grey lines = pseudo stats and blue = non-pseudo?

If so, then it looks like it's working as intended.

Thanks for looking at this.
